src/ir_parser.c

```diff
@@ -0,0 +1,1297 @@
+#include <codegen/intermediate_representation.h>
+#include <ctype.h>
+#include <errno.h>
+#include <error.h>
+#include <ir_parser.h>
+#include <math.h>
+#include <setjmp.h>
+
+/// For isatty().
+#ifdef _WIN32
+#    include <io.h>
+#    define isatty _isatty
+#else
+#    include <stdarg.h>
```

Comment:
This is also needed on Windows (at least using GCC, although IIRC MSVC also has `stdarg.h`)

Reply:
`stdarg.h` is standard C, so it had better!

This is probably just my IDE being too stupid to auto-include headers properly...